pkg/http/api_test.go

```diff
@@ -1408,9 +1392,7 @@ func TestV1DirectMessagingEndpoints(t *testing.T) {
 				mock.MatchedBy(func(b string) bool {
 					return b == "fakeAppID"
 				}),
-				mock.MatchedBy(func(c *invokev1.InvokeMethodRequest) bool {
-					return true
-				}),
+				mock.AnythingOfType("*v1.InvokeMethodRequest"),
```

Comment:
is there a reason to switch this to this format?

Reply:
Uhm, I made a bunch of changes back-and-forth in this code (for a while, I had removed the 2nd argument `appId`), when I reverted them I didn't notice this was changed.

Shouldn't make much of a difference however, the effect is the same.